server/live_query/redis_live_query.go

```diff
@@ -316,14 +320,92 @@ func (r *redisLiveQuery) removeQueryNames(names ...string) error {
 }
 
 func (r *redisLiveQuery) LoadActiveQueryNames() ([]string, error) {
+	r.cache.mu.RLock()
+	if r.cache.cacheExp.After(time.Now()) {
+		names := r.cache.activeQueriesCache
+		r.cache.mu.RUnlock()
+		return names, nil
+	}
+	r.cache.mu.RUnlock()
+
+	cacheCopy, err := r.loadCache()
```

Comment:
agree, this may be a bit overkill in theory but I wanted to simplify the caching pattern to avoid possible race conditions when pulling active queries and sql separately.  The method is primarily used during a live query, so the sql cache populated here is very likely used in the same call.  The only other use of this method is in a cron, where there is indeed some overhead, but only every 15m.